projects/default/controllers/ros/highlevel/WebotsHw.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Description: Webots integration with for `ros_control`.
+
+#include "WebotsHw.hpp"
+
+namespace highlevel {
+
+  WebotsHw::WebotsHw(webots::Robot *robot) : mRobot(robot) {
```

Comment:
I would rename this class `WebotsHW` (with a uppercase W) to match ROS control's `hardware_interface::RobotHW`

Reply:
Webots convention would have led to `WebotsHardware` instead (see https://github.com/cyberbotics/webots/wiki/Coding-Style#cs44-no-abbreviations), but in that case, I believe we should adopt the convention used in the upper class, e..g., `RobotHW`.